client/src/components/chrome/index.css

```diff
@@ -5,6 +5,6 @@
         }
     }
     .container-fluid {
-        padding-top: 83px;
+        padding-top: 93px;
```

Comment:
**facepalm** (no actual issue, just magic numbers changes are...lame, but that's CSS for you)